scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/validation/SampleOverrideTypeProvider.scala

```diff
@@ -98,7 +98,7 @@ final class SampleOverrideTypeProvider extends OverrideTypeProvider {
   ): Unit = ()
 }
 
-class Country(val data: String) extends AnyVal
+class Country(val data: String)
```

Comment:
Cleaning the dependencies revealed a problem here:
if `Country` extends `AnyVal` the compiler can erase the code and use the underlying type.
validation wasn't triggered when the class was erased